content/en/ecosystem/index.md

```diff
@@ -47,13 +47,14 @@ as being part of the Flux Ecosystem.
+| [gimlet-io/capacitor](https://github.com/gimlet-io/capacitor)                                     | Capacitor is a general purpose Flux UI to debug Flux and application issues. Apply [these manifests](https://github.com/gimlet-io/capacitor?tab=readme-ov-file#flux) to install it.                                                                                                                                            |
+| [headlamp/flux-plugin](https://github.com/headlamp-k8s/plugins/tree/main/flux)                    | Headlamp is a Kubernetes UI extensible through plugins. The Flux plugin for Headlamp allows to visualize Flux and perform operations common operations like quick sync'ing, suspending/resuming, and others. See the plugin [README](https://github.com/headlamp-k8s/plugins/tree/main/flux#readme) for installation instructions.                                                                                                                                            |
+| [vmware-tanzu/kubeapps](https://github.com/vmware-tanzu/kubeapps)                                 | Kubeapps is an in-cluster web-based application that enables users with a one-time installation to deploy, manage, and upgrade applications on a Kubernetes cluster. Read [the documentation](https://kubeapps.dev/docs/latest/tutorials/managing-flux-packages) for managing Flux packages using Kubeapps.                    |
+| [weaveworks/vscode-gitops-tools](https://github.com/weaveworks/vscode-gitops-tools)               | GitOps Tools for Visual Studio Code: provides an intuitive way to manage, troubleshoot and operate your Kubernetes environment following the GitOps operating model                                                                                                                                                            |
+| [weaveworks/weave-gitops](https://github.com/weaveworks/weave-gitops)                             | Weaveworks offered a free and open source GUI for Flux under the [weave-gitops](https://web.archive.org/web/20230314183054/https://docs.gitops.weave.works/docs/intro/) project. You can install the Weave GitOps UI using a Flux `HelmRelease`, please see the [get started documentation](https://web.archive.org/web/20230927114646/https://docs.gitops.weave.works/docs/getting-started/intro/) for more details. |
+| [freelensapp/freelens-extension-fluxcd](https://github.com/freelensapp/freelens-extension-fluxcd) | [Freelens](https://freelens.app) is a free and open-source user interface designed for managing Kubernetes clusters. It provides a standalone application compatible with macOS, Windows, and Linux operating systems. This extension visualizes Flux resources and allows to perform operations on them.                      |
```

Comment:
Can you please move this up before tanzu, the list should be alphabetical  ordered, same with the print screen blow.

Reply:
Changed